website/templates/emails/pending_embargo_admin.html.mako

```diff
@@ -6,22 +6,36 @@
     Hello ${user.fullname},<br>
     <br>
     % if is_initiator:
-    You initiated an embargoed registration of ${project_name}. The proposed registration can be viewed here: ${registration_link}.<br>
+    You initiated an embargoed registration of ${project_name} with embargo end date ${embargo_end_date.date()}.
+     The proposed registration can be viewed here: ${registration_link}.<br>
     % else:
-    ${initiated_by} initiated an embargoed registration of ${project_name}. The proposed registration can be viewed here: ${registration_link}.<br>
+    ${initiated_by} initiated an embargoed registration of ${project_name}  with embargo end date ${embargo_end_date.date()}.
+     The proposed registration can be viewed here: ${registration_link}.<br>
     % endif
     <br>
-    If approved, a registration will be created for the project, and it will remain private until it is withdrawn,
-    it is manually made public, or the embargo end date is passed on ${embargo_end_date.date()}.<br>
+    % if is_moderated:
+        If approved, an embargoed registration will be created for the project and sent to ${reviewable.provider.name}
+         moderators for review.
+    % else
+        If approved, a registration will be created for the project, and it will remain private until it is withdrawn,
+         it is manually made public, or the embargo end date is passed on ${embargo_end_date.date()}.
+    % endif
+    <br>
+    Approve this embargoed registration: ${approval_link}.<br>
```

Comment:
These should be html links, not bare urls. `<a href="${approval_link}">Click here</a>`